app/services/api_entreprise_service.rb

```diff
@@ -17,9 +17,19 @@ def fetch_facility_with_siret(siret)
     end
 
     def company_name(company_json)
-      company_name = company_json['entreprise']['nom_commercial']
-      company_name = company_json['entreprise']['raison_sociale'] if company_name.blank?
-      company_name
+      entreprise_hash = company_json.dig('entreprise')
+      return nil unless entreprise_hash
+      company_name = entreprise_hash.dig('nom_commercial')
+      company_name = entreprise_hash.dig('raison_sociale') if company_name.blank?
```

Comment:
It's not the same case, I want `company_name = entreprise_hash.dig('raison_sociale')` to be called only if `entreprise_hash.dig('nom_commercial')` returned a blank string.
If you're speaking of l24, any other way to write this seems more complicated / less understandable.